### DISCUSSION TITLE
Customize the relationship between fan speed and temperatures

### DISCUSSION THREAD

**Discussion by omri09** [BartoszCichecki/LenovoLegionToolkit#1506] [Ideas]
Hello developers,

First of all the software is great.
I would like to suggest adding an option to customize the relationship between fan speed and temperatures, allowing users to control the values. 
For example, set the fans to run at 2670 RPM when the temperature reaches 75°C. Currently these values can't be change.
![image](https://github.com/user-attachments/assets/890ca0b0-cda2-4b2b-8647-d8ac1e23c912)

**Comment by Ace-Radom** [comment]
This has been discussed many times tbh. In one word, it's impossible in current architecture and won't be added. Detailed explaination you can find in #408, #696, #1028 and #1490.
